include/picolibrary/indicator.h

```diff
@@ -36,6 +40,67 @@ enum class Initial_Indicator_State {
     EXTINGUISHED, ///< Extinguished.
 };
 
+/**
+ * \brief Fixed intensity indicator concept.
+ */
+class Fixed_Intensity_Indicator_Concept {
+  public:
+    Fixed_Intensity_Indicator_Concept() = delete;
+
+    Fixed_Intensity_Indicator_Concept( Fixed_Intensity_Indicator_Concept && ) = delete;
+
+    Fixed_Intensity_Indicator_Concept( Fixed_Intensity_Indicator_Concept const & ) = delete;
+
+    ~Fixed_Intensity_Indicator_Concept() = delete;
+
+    auto operator=( Fixed_Intensity_Indicator_Concept && ) = delete;
+
+    auto operator=( Fixed_Intensity_Indicator_Concept const & ) = delete;
+
+    /**
+     * \brief Initialize the indicator's hardware.
+     *
+     * \param[in] initial_indicator_state The initial state of the indicator.
+     *
+     * \return Nothing if indicator hardware initialization succeeded.
+     * \return An error code if indicator hardware initialization failed. If indicator
+     *         hardware initialization cannot fail, return
+     *         picolibrary::Result<picolibrary::Void, picolibrary::Void>.
+     */
+    auto initialize( Initial_Indicator_State initial_state = Initial_Indicator_State::EXTINGUISHED ) noexcept
```

Comment:
Please change `initial_state` (error in issue description) to `initial_indicator_state` for consistency with the parameter documentation and GPIO pin concepts.

```suggestion
    auto initialize( Initial_Indicator_State initial_indicator_state = Initial_Indicator_State::EXTINGUISHED ) noexcept
```